snuba/datasets/table_storage.py

```diff
@@ -31,6 +33,7 @@ def __init__(
         self,
         processor: MessageProcessor,
         default_topic: str,
+        pre_filter: Optional[StreamMessageFilter[KafkaPayload]] = None,
```

Comment:
This class itself is still Kafka specific. We could build an abstract one on top of it. IT would not be used today.